From 6ac6b9f36f975b6903d67c541bd9e1be8196da5f Mon Sep 17 00:00:00 2001
From: Smillick <3185479846@qq.com>
Date: Wed, 14 Oct 2026 12:24:22 +0800
Subject: [PATCH] Declare DLP permission shared IDL outputs

Prepared against the preserved migration baseline; port after prerequisite repairs.

Signed-off-by: Smillick <3185479846@qq.com>
---
 interfaces/inner_api/dlp_permission/BUILD.gn | 1 +
 1 file changed, 1 insertion(+)

diff --git a/interfaces/inner_api/dlp_permission/BUILD.gn b/interfaces/inner_api/dlp_permission/BUILD.gn
index 38abd7f3..8ff5bfa9 100644
--- a/interfaces/inner_api/dlp_permission/BUILD.gn
+++ b/interfaces/inner_api/dlp_permission/BUILD.gn
@@ -16,6 +16,7 @@ import("//build/ohos.gni")
 import("../../../dlp_permission_service.gni")
 
 idl_gen_interface("dlp_permission_interface") {
+  sources_common = [ "DlpPermissionTypes.idl" ]
   src_idl = rebase_path("IDlpPermissionService.idl")
   log_domainid = "0xD005A04"
   log_tag = "DlpPermissionService"
-- 
2.34.1

